hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/ReturnTypes.java

```diff
@@ -58,6 +66,14 @@ private ReturnTypes() {
 
     private static final TupleType RC_ENCODER = TupleType.parse(INT_64);
 
+    public static final String RESPONSE_CODE_CUSTOM_FEES = RESPONSE_STATUS_AT_BEGINNING
+            + FIXED_FEE_V2 + ARRAY_BRACKETS
```

Comment:
how would they do that as the the input parameter (address of the token) is the same.  Our only choice in this instance is to return either the V1 or the V2 version of these results.  I think returning the V2 result is the best we can do?

Reply:
Isn't it dependent on the method signature. Users who already employed V1 bytecode should see V1 structs returned is what I was getting at.
If not could see an unexpected change in behaviour.
This might be a larger versioning topic but either ways we should match mono's current handling of versioning and see if an expansion is necessary.

Reply:
That's just it though - the method signatures for V1 and V2 of `getTokenCustomFees` is the same as the signature only takes into account the name of the function and the parameters - and not the return types.  I will revert to using V1 as that is what is happening in mono service.  V2 is only used when creating the token.  This could cause problems in the future but as you say this could necessitate a larger discussion.